packages/report-server/.env.example

```
@@ -10,3 +10,4 @@ DHIS_USERNAME=
 JWT_SECRET=
 PORT=
 API_CLIENT_SALT=
+ENTITY_SERVER_API_URL=
```

I didn't change the name to `ENTITY_SERVER_API_URL` to be consistent with the existing convention in `psss-server`. Ideally we'd make all such variables consistent across packages!

```suggestion
ENTITY_API_URL=
```

---

Yup, I'll change this but it's also `ENTITY_SERVER_API_URL` in `lesmis-server` as well...

---

True, as discussed in https://github.com/beyondessential/tupaia/pull/2382#discussion_r612213598

For now being consistent across the package should be fine, the next step would be ensuring cross-package consistency